README.md

```diff
@@ -36,8 +36,6 @@ panic=abort`, for your target.
 $ cargo install xargo
 ```
 
-But we also have [binary releases] for the three major OSes.
-
 [binary releases]: https://github.com/japaric/xargo/releases
```

Comment:
This is a dead Markdown reference now, is it not?

Reply:
Fixed.